manim/animation/animation.py

```diff
@@ -22,6 +22,68 @@
 
 
 class Animation:
+    """An animation.
+
+    Animations have a fixed time span.
+
+    Parameters
+    ----------
+    mobject
+        The mobject to be animated. This is not required for all types of animations.
+    lag_ratio
+        Defines the delay after which the animation is applied to submobjects. This lag
+        is relative to the duration of the animation.
```

Comment:
Already really nice!
Reading this, I don't completely get, what the word "ratio" means in the context. Means the ratio of 1, that there is no overlap of the submobjects?
And something like "ratio" =    "overlap of two subanimation"  /  "length of one subanimation" ?
I know this formula is wrong, but I did not have the time right now to properly think about it ;)

Reply:
I can see where you're coming from - I was struggling to write that description and was never quite happy with it. My approach to solve that problem was to add an example. Imo the combination explains it reasonably well without having a too wordy explanation.

Reply:
Ture.
There can definitely be fine-tuning, but much better than nothing :)